core/controllers/questions_list.py

```diff
@@ -89,3 +89,30 @@ def get(self, comma_separated_skill_ids):
             'next_start_cursor': next_start_cursor
         })
         self.render_json(self.values)
+
+
+class QuestionCountDataHandler(base.BaseHandler):
+    """Provides data regarding the number of questions assigned to the given
+    skill ids.
+    """
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.open_access
+    def get(self, comma_separated_skill_ids):
+        """Handles GET requests."""
+        skill_ids = comma_separated_skill_ids.split(',')
+
+        for skill_id in skill_ids:
+            try:
+                skill_domain.Skill.require_valid_skill_id(skill_id)
+            except Exception:
+                raise self.PageNotFoundException(Exception('Invalid skill id'))
+
+        question_count = question_services.get_question_count_by_skill_ids(
```

Comment:
Perhaps by --> for, and add "total" before "question count". Otherwise it sounds like you're getting multiple values. 

Also perhaps call the variable name total_question_count, here and below, to underscore that you're getting a total.

Reply:
Done.